video_xblock/static/js/player_context_menu.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * This part is responsible for the player context menu.
+ * Menu Options include:
+ * - Play / Pause
+ * - Mute / Unmute
+ * - Fill browser / Unfill browser
+ * - Speed
+ *
+ */
+
+domReady(function() {
+
+  var videoPlayer = document.getElementById("{{ video_player_id }}");
+  var dataSetup = JSON.parse(videoPlayer.getAttribute('data-setup'));
+  var playbackRates = dataSetup.playbackRates;
+  // VideoJS Player() object necessary for context menu creation
+  var player = videojs('{{ video_player_id }}');
+
+  // Delegate creation of a nested submenu for a context menu
+  $(videoPlayer).on('mouseenter', 'li.vjs-menu-item:last', function(e) {
```

Comment:
Event delegation seems not to work correctly if an event is bound to a videojs object. Would prefer to re-write with pure JavaScript in the next PR.